terraform.tfstate

```diff
@@ -0,0 +1,9 @@
+{
+  "version": 4,
+  "terraform_version": "1.5.7",
+  "serial": 1,
+  "lineage": "9c8656a6-ef72-9bec-3d61-d2b72f6ba310",
+  "outputs": {},
+  "resources": [],
+  "check_results": null
+}
```

Comment:
This file shouldn't exist since the terraform state is stored in S3, right? We certainly don't want to commit the state here by accident.